### PR TITLE
Refactoring the DetailedObjects view

### PR DESCRIPTION
**Pull request by rpmoore**

### CODE REVIEW

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java [hunk 1]
```diff
@@ -38,8 +38,8 @@
 import java.util.Map;
 import javax.annotation.Nullable;
 
-public class GetFilteredObjects extends CliCommand<GetDetailedObjectsResult> {
-    private final static org.slf4j.Logger LOG = LoggerFactory.getLogger(GetFilteredObjects.class);
+public class GetDetailedObjects extends CliCommand<GetDetailedObjectsResult> {
+    private final static org.slf4j.Logger LOG = LoggerFactory.getLogger(GetDetailedObjects.class);
```
Comment (berkeleysquare) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java]: Command name change should be updated in resources: com.spectralogic.ds3cli.command.CliCommand and com.spectralogic.ds3cli.help.properties

Comment (berkeleysquare) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java]: Command name change should be updated in resources: com.spectralogic.ds3cli.command.CliCommand and com.spectralogic.ds3cli.help.properties

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjectsPhysical.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
  * Use the DetailedObjectsPhysicalView to format output
  * by physical placement: one line for each instance on tape
  */
-public class GetFilteredObjectsPhysical extends GetFilteredObjects {
+public class GetDetailedObjectsPhysical extends GetDetailedObjects {
```
Comment (berkeleysquare) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjectsPhysical.java]: Command name change should be updated in resources: com.spectralogic.ds3cli.command.CliCommand and com.spectralogic.ds3cli.help.properties

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java (hunk 1 above)
Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java]: It looks like only the help.properties file needed to be changed.  The CliCommand file already had the change.

**Pull request merged**